### DISCUSSION TITLE
Achievements issue

### DISCUSSION THREAD

**Discussion by ArmSershant** [community/community#124752] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

![image](https://github.com/community/community/assets/101866789/2558aabd-cbd4-4a9c-8d81-714026b19f4a)
I have done many commits and done for example a pull request, I created a repo, I even worked in an organization, I was a contributor, but I didn't get any achievement for it, but on many people's pages I see that to do the same things they have received an achievement, is that a bug or something else ?

**Comment by LelouchFR** [comment]
as I see, it doesn't seem you opened any pull request on a public repository. (as your gh stats says it)

![](https://github-readme-stats.vercel.app/api?username=armsershant&show_icons=true&locale=en)

**Reply by ArmSershant** [reply]
they are in private repos

**Comment by CopperEagle** [comment]
Hey @ArmSershant 

I do not see any pull request in your activity history. Not quite sure what is going on.

To test, you may want to try to get a simple achievment, like "Quickdraw". If you need guidance on how to get the different achievments, have a look at this [repository](https://github.com/4xmen/Get-Github-Achievements).

**Reply by ArmSershant** [reply]
Hey @CopperEagle I've done many pr's, but in private repos

**Reply by CopperEagle** [reply]
Ah, okay @ArmSershant . Then you need to make sure your profile settings look like this:
![Settings](https://github.com/community/community/assets/137458753/2addd65e-be6f-4c78-80f7-10a9441e81ee)

I am guessing that the second option is turned off in your profile right now. You need to turn it on.
The repositories' names will not be revealed, no worries.

**Reply by CopperEagle** [reply]
[Here](https://github.com/davecheney) is an example of a profile that has the second option turned on. As you can see, there is the note about commits to private repositories in the activity log. However, it does not tell the name or anything. The option must be turned on to allow achievments to count towards these repositories as well.

**Reply by ArmSershant** [reply]
Everything okay with pr's, but for example there is an achievement for creating a repo, as you see I have 40 repos (more in private), why I didn't get that achievement?

**Reply by CopperEagle** [reply, marked as answer]
AFAIK, such an achievment does not exist. [Here](https://github.com/4xmen/Get-Github-Achievements) is a list of all and how to get them.
